worker/utils/mongo.js

```diff
@@ -1,4 +1,6 @@
 const { MongoClient } = require('mongodb');
+const request = require('request');
```

Comment:
mongo.js is a utility class for accessing the mongoDB database -- what architecture concept might we use to insulate the caller (in this case githubpushJob.js) from the implementation details? Hint there is another class in this directory that would be a nice home for calls to encapsulated database calls.